webpack.config.js

```diff
@@ -66,6 +64,24 @@ var config = {
       }
     ]
   },
+  resolve: {
+    root: path.resolve(__dirname),
+    alias: {
+      util: 'htdocs/js/util',
+      jsx: 'jsx',
+      Breadcrumbs: 'jsx/Breadcrumbs',
+      DynamicDataTable: 'jsx/DynamicDataTable',
+      FilterForm: 'jsx/FilterForm',
+      Form: 'jsx/Form',
+      Markdown: 'jsx/Markdown',
+      PaginationLinks: 'jsx/PaginationLinks',
+      Panel: 'jsx/Panel',
+      ProgressBar: 'jsx/ProgressBar',
+      StatidDataTable: 'jsx/StatidDataTable',
```

Comment:
This particular addition is used to allow people to components such as `StaticDataTable` as
`import StaticDataTable from 'StaticDataTable';` instead of specifying the right path, but you are totally right about the typo